Pod/Classes/Keyboard/Keyboard.swift

```diff
@@ -33,7 +33,7 @@ import UIKit
 /**
  *  A set of abstractions around the keyboard.
  */
-final class Keyboard {
+open class Keyboard {
```

Comment:
@dostrander is `public` sufficient for the amount of opening up you were thinking of, or do you have a reason it needs to be `open`?